### PR TITLE
Bug fixed for Weather app

### LINKED ISSUE #106: Weather App: JS
**Describe the bug**
Icons missing

**Screenshots**
image

@shruti-sen2004

### PR DESCRIPTION
**Pull request by shruti-sen2004**
#106 
Title : Bug fixed for missing weather icons

Issue No. : 106

Code Stack :  Js

Close #106 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

**Comment by sadanandpai** [comment]
@shruti-sen2004 I believe the issue was due to missing icon for **_Haze_** weather condition

Haze.png needs to be added

**Comment by shruti-sen2004** [comment]
okay .....wait let me add that

File: native/machine-coding/weather-app/index.js [hunk 1]
```diff
@@ -20,7 +20,21 @@ async function checkWeather(city) {
     document.querySelector('.humidity').innerHTML = data.main.humidity + '%';
     document.querySelector('.wind').innerHTML = data.wind.speed + 'km/h';
 
-    weatherIcon.src = `images/${data.weather[0]?.main?.toLowerCase()}.png`;
```
Comment (sadanandpai) [inline native/machine-coding/weather-app/index.js:23]: There is no problem with the code probably

Comment (shruti-sen2004) [reply native/machine-coding/weather-app/index.js:23]: okay

**Comment by shruti-sen2004** [comment]
I have added haze.png and also reverted back to the previous code for loading the images

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**